Helper/Hashcash/Token.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace EMS\ClientHelperBundle\Helper\Hashcash;
+
+
+class Token
+{
+    const DELIMITER = '|';
```

Comment:
I think the general rule is to put the consts of a class after the properities, check with @Davidmattei

Reply:
ok